package.json

```diff
@@ -89,7 +91,10 @@
 		"webpack-cli": "^3.3.12"
 	},
 	"dependencies": {
+		"bootstrap": "^4.5.3",
+		"bootstrap-fork": "^3.3.6",
```

Comment:
Are these dependencies needed? I don't see them referenced anywhere in the project.

Reply:
@r1cebank Don't think so! Sorry, they must have been added by npm for some reason...

Reply:
No worries, thanks for the contribution!

Reply:
Can you remove the changes for the package.json and package-lock.json and i will merge this.